### DISCUSSION TITLE
Still not being possible to use Deno.readTextFileSync on deno deploy

### DISCUSSION THREAD

**Discussion by adeirjunior** [denoland/deno#18201] [Q&A] [answered]
I'm creating a module to create multilingual websites on Deno. I was using Deno.readTextFileSync to get the JSON file with the different languages, and everything worked as it should. However, when I deployed it on a test website in Deno Deploy, an error showed up saying that Deno.readTextFileSync is not a function.

It looks like, for some reason, it is blocked on deployment, which is really sad because it is way harder to use Deno.readTextFile since it is a Promise.

**Comment by iuioiua** [comment, marked as answer]
`Deno.readTextFileSync()` is not supported on the Deno Deploy Runtime API, but `Deno.readTextFile()` is ([link](https://deno.com/deploy/docs/runtime-api)).

**Reply by adeirjunior** [reply]
Thank you for your answer, I will end up using ``Deno.readTextFile()`` in the end

**Comment by adeirjunior** [comment]
``Deno.readTextFileSync()`` is not supported on the Deno Deploy Runtime API
